core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -184,21 +189,23 @@ trait EntityComponent {
     }
 
     def entityAttributes(entityId: Rep[Long]): EntityAttributeQuery = for {
-      entityAttrRec <- entityAttributeQuery if entityAttrRec.ownerId === entityId
+      entityAttrRec <- entityAttributeQuery if entityAttrRec.ownerId === entityId && ! entityAttrRec.deleted
```

Comment:
`entityAttrRec` works like a function parameter here, so this should be equivalent to `entityAttributeQuery.filter(_.owner_id === entityId).filterNot(_.deleted)`